packages/dataviews/src/dataform-controls/toggle-group.tsx

```diff
@@ -34,7 +43,28 @@ export default function ToggleGroup< Item >( {
 			( el ) => el.value === value
 		);
 		return (
-			<ToggleGroupControl
+			<ValidatedToggleGroupControl
```

Comment:
@mirka Unless I'm doing something wrong, it looks like the required validation is not working for the ValidatedToggleGroupControl. It should be triggered when:

- the control starts with no value selected
- it gets focus and then loses it

The same happens in the [component story](https://github.com/WordPress/gutenberg/blob/7ee2a7aca4979f984958ee70f4096f1c2b56738f/packages/components/src/validated-form-controls/components/stories/toggle-control.story.tsx#L41).

Do you have any thoughts here (feel free to push to this PR if that helps)?

Reply:
Thanks for the report. Should be fixed as of b3eb849. (It was a small workaround for an upstream bug, but that was already fixed in #69969.)

Reply:
In testing, it looks like if I tab to the control and tab away without selecting anything, it appears to correctly fire the validation. The validation doesn't appear to fire on the initial load of the story. That sounds fine to me, as the label indicates it's a required field, but just thought I'd double check that's what you were expecting, André?